.sass-lint.yml

```diff
@@ -6,6 +6,7 @@ files:
 
 
 rules:
+  class-name-format: 0
```

Comment:
Not sure. Doesn't look like **sass-lint** currently supports disabling rules per-block or per-file (see https://github.com/sasstools/sass-lint/issues/70), so we either have to disable the rule globally here, or just try and ignore the DebugPage.scss using a glob or something.